commit 451cce0f870ce5ce484cd67bb15bba5c4833af03 (refs/changes/69/469/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-09-15T14:10:10-07:00 (3 years, 1 month ago)
    
    MORAY-358 use a livelier url to reference node-fast
    
    The canonical upstream is now in the joyent org.

diff --git a/README.md b/README.md
index a5981fb..94531de 100644
--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 This repo contains Moray, the highly-available key/value store from Joyent.
 Moray offers a simple put/get/del (as well as search) protocol on top of
-Postgres 9.x, over plain TCP see <https://github.com/mcavage/node-fast>.
+Postgres 9.x, over plain TCP see <https://github.com/joyent/node-fast>.
 
 This repository is part of the Joyent SmartDataCenter project (SDC), and the
 Joyent Manta project.  For contribution guidelines, issues, and general
